test/testnet/localnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Remote Version of Localnet
+
```

Comment:
can we link to this file from the docs, e.g., this file: https://github.com/tendermint/tendermint/blob/master/docs/networks/README.md

Reply:
Will do 👍